backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/PickingCandidateRepository.java

```diff
@@ -512,6 +512,35 @@ public boolean isHuIdPicked(@NonNull final HuId huId)
 				.anyMatch();
 	}
 
+	@NonNull
+	public ImmutableList<PickingCandidate> getDraftedByHuIdAndPickingSlotId(
+			@Nullable final HuId huId,
+			@Nullable final PickingSlotId pickingSlotId)
+	{
+		if (huId == null && pickingSlotId == null)
+		{
+			return ImmutableList.of();
+		}
```

Comment:
why not throwing an error in this case?

Reply:
I was thinking to let the repo method as lenient as possible, but sure, I can thorw the error